campair/campair/View/Editor/Detail/EditorDetailView/EditorDetailContentsVersionView.swift

```diff
@@ -7,8 +7,8 @@
 
 import SwiftUI
 
-struct EditorDetailView: View {
-    @StateObject var editorDetailContent: EditorDetailViewModel
+struct EditorDetailContentsVersionView: View {
+    @ObservedObject var editorDetailContent: EditorDetailViewModel
```

Comment:
```suggestion
struct EditorDetailContentsVersionView: View {
    @ObservedObject var editorDetailContent: EditorDetailViewModel
```
Why do you changed StateObject to ObsevedObject? Is there any reason?

Reply:
I changed it to ObservedObject because EditorDetailView, which is the top view of this view, receives it as a StateObject and hands it over!
<img width="680" alt="스크린샷 2022-06-15 오후 8 22 04" src="https://user-images.githubusercontent.com/82457928/173815927-ab6622f7-6c6f-4b71-b30a-27aed5ee74ad.png">

Reply:
GOOD!!  👌